Classes/Utility/ImageVariantsUtility.php

```diff
@@ -29,22 +30,27 @@ class ImageVariantsUtility
     protected static $defaultVariants = [
         'default' => [
             'breakpoint' => 1200,
-            'width' => 1100
+            'width' => 1100,
+            'widthRetina' => 2200
         ],
         'large' => [
             'breakpoint' => 992,
-            'width' => 920
+            'width' => 920,
+            'widthRetina' => 1840
         ],
         'medium' => [
             'breakpoint' => 768,
-            'width' => 680
+            'width' => 680,
+            'widthRetina' => 1360
         ],
         'small' => [
             'breakpoint' => 576,
-            'width' => 500
+            'width' => 500,
+            'widthRetina' => 1000
```

Comment:
Please remove default variants, we only need to enable the user to configure retina images.
I would not like to deliver them by default.